Vagrantfile

```diff
@@ -4,8 +4,16 @@
 # Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
+# Software version variables
+GOVERSION = "1.7.4"
```

Comment:
This is the ruby variable, and then it's defined again lower in the shell script that vagrant runs. on the box. It's slightly redundant, but it allows the variables to be grouped together. I could however pull out the shell script variable, and just replace all of those with the ruby variable if you think that would be cleaner.